govuk-one-login/express-openid-connect-prototype
# Generalise references to SPOT and improve documentation

# What
- 0037ea8 - add example .env file as `example.env.txt` to help users configure their client
- 04891cf - remove references to SPOT which is internal and make more general
- 311e617 - improve private key example to make it more explicit
- ba763a7 - improve `README.md`
  - clarify it uses the prototype kit
  - make it clear up front this uses node.js
  - explain the use of of dotenv module to read config
  - provide example configuration in `example.env.txt`
  - provide example base url
  - not obviously, obviously 😬

# How to review
- clone locally 
- ensure README.md still makes sense
- npm `install && npm run dev`
- make sure it still works after renaming the variables
